### ISSUE TITLE
Windows builds do not have correct version number

### ISSUE THREAD

**Issue by MeirBon** [phonegap/phonegap-cli#725]
### PhoneGap CLI, node & npm versions
- phonegap-cli: 6.5.0
- node: 6.10.3
- npm: 5.0.3
### Expected Behaviour
When building an app and specifying the following in config.xml:
```
<widget xmlns="http://www.w3.org/ns/widgets"
        xmlns:gap="http://phonegap.com/ns/1.0"
        windows-packageVersion="2.0.47"
        version="2.0.47"
        versionCode="2047000">
```
we expect to build a 2.0.47 version of the app.
### Actual Behaviour
A version 2.0.47 build is created for Android and iOS, but for Windows 10 a 2.0.4.0 build is created instead of a 2.0.47. The build app works, but versioning is wrong. Adding or removing windows-packageVersion does not change the outcome.

This issue is also described in the [forums](https://forums.adobe.com/message/9247410#9247410).
### Steps to Reproduce 
Build an app with major/minor/patch versioning. Apps for Android and iOS will be build as expected, Windows apps will have a wrong version number.
#### Sample Code that illustrates the problem
See our configuration in 'Expected Behaviour'.
#### Error messages
Builds don't produce any errors.

**Comment by surajpindoria** [comment]
Hey @MeirBon,

Sorry for the late reply, I started looking into this and then was running into network issues as I was testing. Are you seeing this error when using the CLI or from Build?

**Comment by MeirBon** [comment]
Hi @surajpindoria,

No problem, thanks for looking into this. At the time of creating this issue there was no error from both the CLI and Build. I also opened an issue in [phonegap/build](https://github.com/phonegap/build/issues/597) where I was told it was supposed to be fixed. But where apps would be build before this 'fix', they now produce the following error:
```
COMPILE OUTPUT
--------------------------------------------------------------------------------
CertUtil: -importPFX command FAILED: 0x80070002 (WIN32: 2 ERROR_FILE_NOT_FOUND)
CertUtil: The system cannot find the file specified.
My "Personal"
CertUtil: -delstore command completed successfully.
```
At first the builds would come through fine, but would have wrong versioning (e.g. 2.0.4 instead of 2.0.47). Now any build we run fails.

**Comment by surajpindoria** [comment]
Did you manage to get this resolved? @MeirBon

**Comment by MeirBon** [comment]
@surajpindoria Sort of. Cordova Windows 4.4.3 has some bugs with image sizes that it supports (4.4.3 only accepts WP8 images, so we end up with Cordova images in Windows 10 builds..). This seemed to be fixed in 5.0.0, but I forgot to check if Phonegap actually supports 5.0.0. 
So we figured out why building wasn't working, but we can't distribute our apps yet due to bugs in Cordova. I guess we will just have to wait for support for the new Cordova version.
